openshift/redash/charts/redis-exporter/Chart.yaml

```diff
@@ -0,0 +1,5 @@
+---
+apiVersion: v2
+description: Redis and exporter master/slave templates based on Statefulsets
```

Comment:
we might want to change the expression here to Primary/Replica or Primary/Secondary